React/Views/RCTHandledKey.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#if TARGET_OS_OSX // [macOS
```

Comment:
The convention we've followed for macOS files is to leave a single `// [macOS]` comment at the top. Though, if this is unclear, I'd be happy to revisit it

https://github.com/microsoft/react-native-macos/blob/main/docs/DiffsWithUpstream.md#forked-files

Reply:
Thanks for the link, updated.